doc/source/whatsnew/v1.0.0.rst

```diff
@@ -174,6 +174,10 @@ Backwards incompatible API changes
 
    pd.arrays.IntervalArray.from_tuples([(0, 1), (2, 3)])
 
+- Changed the default configuration value for ``options.matplotlib.register_converters`` from ``True`` to ``"auto"`` (:issue:`18720`).
+  Now, pandas custom formatters will only be applied to plots created by pandas, through :meth:`~DataFrame.plot`.
+  Previously, pandas' formatters would be applied to all plots created *after* a :meth:`~DataFrame.plot`.
```

Comment:
Add a reference to the longer section below?

Reply:
I meant the section below in the whatsnew (but a link to the docs is actually also useful). There are now 2 entries for this change, and it is not really clear to me how they relate to each other (the one seems to explain it in more detail)

Reply:
Oh, whoops.